src/dtw.jl

```diff
@@ -12,6 +12,13 @@ function nanopore_dtw(x::Vector{Float32}, y::Vector{Float32},
     return dtw_bhattacharyya(kmerdists1, kmerdists2)
 end
 
+
+function nanopore_multi_dtw(x::Vector{Vector{Float32}},
+     kmerpath::String = "models/r9.4_70bps.u_to_t_rna.5mer.template.model")
+
+    return dtw_multiple_cost_matrix_bhattacharyya(x)
```

Comment:
why it is dtw_multiple_cost_matrix... instead of paralel to our dtw for 2 sequences?
dtw_multi_bhattacharyya

Reply:
point being why not follow the same sheme as before